### PR TITLE
Feature/t124050/licenses

### PR DESCRIPTION
**Pull request by coreyfloyd**

### CODE REVIEW

**Comment by bgerstle** [comment]
:100: :rocket: :sparkles: :rainbow: 🎇

**Comment by bgerstle** [comment]
bravo

**Comment by montehurd** [comment]
:+1: 

Side-note: I tapped on the Gerrit link and the repo mirroring seems to be down.

File: Wikipedia/Code/AboutViewController.m [hunk 1]
```diff
@@ -270,8 +208,11 @@ - (BOOL)webView:(UIWebView*)webView shouldStartLoadWithRequest:(NSURLRequest*)re
     NSURL* requestURL = [request URL];
 
     if ([[self class] isLicenseURL:requestURL]) {
-        NSString* licenseText = [self licenseTextForLicenseURL:requestURL];
-        [self.webView loadHTMLString:licenseText baseURL:nil];
+        VTAcknowledgementsViewController* vc = [VTAcknowledgementsViewController acknowledgementsViewController];
+        vc.headerText = NSLocalizedString(@"We love open source software <3.", nil);
```
Comment (montehurd) [inline Wikipedia/Code/AboutViewController.m]: Add to strings file?

Comment (coreyfloyd) [inline Wikipedia/Code/AboutViewController.m]: @montehurd oh i forgot i eft that in there i was just copying sample code

Comment (coreyfloyd) [inline Wikipedia/Code/AboutViewController.m]: @montehurd @bgerstle doc you guys want to write anything there or should we just make it blank?

Comment (bgerstle) [inline Wikipedia/Code/AboutViewController.m]: also, use heart emoji kplzthx

Comment (montehurd) [inline Wikipedia/Code/AboutViewController.m]: lol

Comment (montehurd) [inline Wikipedia/Code/AboutViewController.m]: I'm fine with just leaving it "We love open source software" (w/o the <3 part, which TWN may push back on)

**Comment by montehurd** [comment]
screen shot 2016-02-24 at 6 29 34 pm

**Pull request merged**